cmd/draft/connect.go

```diff
@@ -43,7 +47,7 @@ func newConnectCmd(out io.Writer) *cobra.Command {
 	f := cmd.Flags()
 	f.Int64Var(&cc.logLines, "tail", 5, "lines of recent log lines to display")
 	f.StringVarP(&runningEnvironment, environmentFlagName, environmentFlagShorthand, defaultDraftEnvironment(), environmentFlagUsage)
-	f.StringVarP(&containerName, "container", "c", "", "name of the container to connect to")
```

Comment:
oh man I totally think we should bring the container flag back! I think it's a nice addition for people who don't care about ports but do care about only connecting to one container in their pod like a sidecar container. What do you think?

Reply:
Sure, I'm open to it !
The only question is how to handle both `-c` and `-p` flags.

Reply:
We'll have to parse the container flag first. We'll need to find which ports are open on that container and decide which local ports to forward based on which ones are open on that container. That logic should probably live in `pkg/draft/local`. What do you think?